build_tools/ci/cpu_comparison/run_test.py

```diff
@@ -0,0 +1,471 @@
+# Copyright 2024 The IREE Authors
```

Comment:
This script is hopefully 1:1 equivalent with the run_test.sh file which is removed in this PR